docs/_menu_template.html

```diff
@@ -0,0 +1,4 @@
+<li class="nav-header">
```

Comment:
I also don't think you should add these templates for the docs of this repository.
We are proposing an alternative for the default, so having a template that mimics the default is a bit strange.